### PR TITLE
PIM-5900: Lifting UI

### PR DESCRIPTION
**Pull request by willy-ahva**
Here is the render of the main menu :
akeneo main menu lifted

Here the render of the new grid :

lifting grid ui

Updates correspond to the Page 2, 3, 7 and 8 of the AUDIT UX & UI DESIGN in the Jira.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | n
| Added Behats                      | n
| Changelog updated                 | n
| Review and 2 GTM                  | 
| Micro Demo to the PO (Story only) | n
| Migration script                  | n
| Tech Doc                          |

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/css/less/oro.less [hunk 1]
```diff
@@ -49,7 +49,7 @@ a:focus {
 }
 .dropdown-menu {
   margin-top:-1px;
-  .border-radius(3px);
+  .border-radius(0px);
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Resources/public/css/less/oro.less]: 0

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less [hunk 2]
```diff
@@ -140,29 +138,21 @@ header {
 #oroplatform-header {
   .oroplatform-menu {
     > li.dropdown {
-      margin: 0 3px;
+      margin: 0px;
       z-index: 900;
       > a {
-        margin: 6px 0 6px;
-        padding: 4px 10px;
+        border-radius: 0px;
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: please use the mixin `.border-radius(0);`

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less [hunk 3]
```diff
@@ -140,29 +138,21 @@ header {
...
+        padding: 0px 15px;
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: "0px" should not be accepted by cs fixer

File: src/Pim/Bundle/UIBundle/Resources/public/css/less/variables.less [hunk 4]
```diff
@@ -87,7 +87,7 @@
 @warningText:       @yellowDarker;
 
 // Purples
-@akeneoPurple:        #9452ba;
+@akeneoPurple:        #562482;
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/css/less/variables.less]: Please use defined Purple in guidelines: #52267D

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less (hunk 2 above)
Comment (jjanvier) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: yes please

File: src/Pim/Bundle/UIBundle/Resources/views/Default/logo.html.twig [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <h1 class="logo">
     <a href="{{ path('oro_default') }}" title="{{ oro_config_value('oro_ui.application_title') }}">
-        <img src="{{ asset('/bundles/pimui/images/logo.png') }}" alt="{{ oro_config_value('oro_ui.application_name') }}">
+        <img src="{{ asset('/bundles/pimui/images/logo.jpg') }}" alt="{{ oro_config_value('oro_ui.application_name') }}">
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/views/Default/logo.html.twig]: I think png was a better format for this image :)

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less [hunk 6]
```diff
@@ -44,7 +43,7 @@
     }
     .logo {
       position: relative;
-      height: 40px;
+      height: 50px;
```
Comment (jjanvier) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: seems weird, are you sure about that?

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less [hunk 7]
```diff
@@ -172,24 +162,21 @@ header {
...
-        .border-radius(0 3px 3px 3px);
         width: auto;
         padding: 0;
         font-size: @font-size-14;
         .box-shadow(0 2px 3px rgba(0, 0, 0, 0.3));
         &:after,
         &:before { border:none; }
         li {
-          line-height: 1.3em;
+          line-height: 44px;
```
Comment (jjanvier) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: align please

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less [hunk 8]
```diff
@@ -201,23 +188,19 @@ header {
...
       left: auto;
-      top: 39px;
+      top: 45px;
     }
   }
 }
 #oroplatform-header {
   div.navbar-inner {
-    background: @akeneoPurpleDark;
+    background: @akeneoPurple;
```
Comment (jjanvier) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: I would have used composition here

File: src/Pim/Bundle/UIBundle/Resources/public/css/pinbar.less [hunk 9]
```diff
@@ -7,7 +7,7 @@
   padding: 0 30px 0 0;
   position: relative;
   z-index: 998;
-  background-color: @akeneoPurpleDarker;
+  background-color: @akeneoPurpleDark;
```
Comment (jjanvier) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pinbar.less]: you don't respect the SOLID principles

File: src/Pim/Bundle/UIBundle/Resources/views/Default/logo.html.twig (hunk 5 above)
Comment (jjanvier) [inline src/Pim/Bundle/UIBundle/Resources/views/Default/logo.html.twig]: don't forget KISS

File: src/Pim/Bundle/UIBundle/Resources/public/css/less/variables.less (hunk 4 above)
Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/css/less/variables.less]: Color I put is the one from the audit. Don't know which one we have to choose.

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less (hunk 6 above)
Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: yes

**Comment by jjanvier** [comment]
GTM if you find the enigma of Father Fouras:
>On le demande,
>On le pours>it,
>On le perd,
>On le passe.

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less (hunk 8 above)
Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: I prefered Decorator & Singleton pattern compiled by a MVVM model abstracter.

File: src/Pim/Bundle/UIBundle/Resources/public/css/header.less (hunk 7 above)
Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/css/header.less]: Noob, it's already align. But maybe with low-angle shot, it's not. :trollface:

File: src/Pim/Bundle/UIBundle/Resources/public/css/pinbar.less (hunk 9 above)
Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pinbar.less]: I respected FLABBY principle.

**Comment by pierallard** [comment]
GTM

**Comment by nidup** [comment]
❤

**Pull request merged**